app/components/questions.js

```diff
@@ -0,0 +1,108 @@
+// https://github.com/EnCiv/undebate-ssp/issues/48
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import CountLimitedTextInput from './count-limited-text-input'
+import Submit from './submit'
+
+export default function Questions({ electionOM, onDone }) {
+    const classes = useStyles()
+    const { electionMethods } = electionOM
+    const [questions, setQuestions] = useState([''])
```

Comment:
We need to consider that their may be initial data in the questions. Something like
```
const [questions, setQuestions]=useState(electionObj.questions)
```
But then there's another issue- electionObj.questions in an object, not an array.   (See #46)
You can map it into an array here, or you can try to use it as an object in the rest of the code.  Before you decide - see the rest of the comments.

Reply:
Also make sure to consider that electionObj.questions may be undefined initially.  so useState(electionObj.questions || {0: ''} )